cli/ioctl/cmd/account/accountbalance.go

```diff
@@ -33,7 +34,11 @@ var accountBalanceCmd = &cobra.Command{
 
 // balance gets balance of an IoTeX blockchain address
 func balance(args []string) (string, error) {
-	address, err := alias.Address(args[0])
+	addr, err := config.GetAddress(args)
```

Comment:
```suggestion
	var addr string
	var err error
	if len(args) == 0 {
	    addr, err = config.GetAddress()
	} else {
	    addr, err = alias.Address(addr)
	}
```

Also,

1. make `config.GetAddress` as simple as fetching the address from config
2. See if the code above could be a common util method